app/queries/most_recently_updated_file_sets.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+class MostRecentlyUpdatedFileSets
+  def self.queries
+    [:most_recently_updated_file_sets]
+  end
+
+  attr_reader :query_service
+  delegate :resource_factory, to: :query_service
+  delegate :orm_class, to: :resource_factory
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def most_recently_updated_file_sets
+    run_query(query)
+  end
+
+  def query
+    <<-SQL
+      select * FROM orm_resources WHERE
+      internal_resource='FileSet'
+      order by updated_at desc
```

Comment:
I think it's probably worth combining the impl here with `LeastRecentlyUpdatedFileSets` — and maybe asking whether they should be a single class with a parameter to flip the ordering.

Reply:
Can I do that like `order by updated_at ?` and a sql substitution? I tried doing something like that with a `limit` and it didn't seem to accept the `?`.

Reply:
I don't know if you can do it with SQL interpolation, but you couldn't you put the order statement in a separate method and concatenate them?

Reply:
ah, good point; it's just a string isn't it :D okay i can work something out here